src/arche/rules/coverage.py

```diff
@@ -98,3 +100,40 @@ def compare_scraped_fields(source_df: pd.DataFrame, target_df: pd.DataFrame) ->
         result.add_info(f"New - {', '.join(new_fields)}")
 
     return result
+
+
+def anomalies(jobs: List[Job]) -> Result:
+    """Find fields with significant deviation. Significant means `> 2 * std()`
+
+    Args:
+        jobs: a list of jobs (a sample). The last job is where deviation is checked.
+        It's not included in the sample.
+
+    Returns:
+        A Result with a dataframe of significant deviations
+    """
+    result = Result("Anomalies")
+    raw_stats = [job.items.stats() for job in jobs]
+
+    counts = (
+        pd.DataFrame(rs.get("counts") for rs in raw_stats)
+        .fillna(0)
+        .drop(columns="_type")
+    )
+    items_len = [rs["totals"]["input_values"] for rs in raw_stats]
+    sample = counts.apply(lambda x: x / items_len)
+    keys = [job.key for job in jobs]
+    sample.index = keys
+    sample.rename(index={keys[-1]: "last"}, inplace=True)
+    sample.loc["mean"] = sample.loc[keys[:-1]].mean()
+    sample.loc["std"] = sample.loc[keys[:-1]].std()
+    sample = sample.T
+    sample["last std"] = sample["last"] - sample["mean"]
```

Comment:
maybe `deviation` is better than `last std`

Reply:
Right, I confused standard deviation here with just deviation